AWS/aws-waf/ingest/parser.yml

```
@@ -0,0 +1,59 @@
+name: aws-waf
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+  - name: parse_date
+    external:
+      name: date.parse
+      properties:
+        input_field: '{{json_event.message.timestamp | int}}'
+        output_field: datetime
+        format: timestamp
+
+  - name: set_ecs_fields
+  - name: set_terminating_Rule_Match_Details
+
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          '@timestamp': '{{parse_date.datetime}}' 
+          event.kind: 'event'
+          event.category: ['network']
+          event.type: ['access']
+          event.dataset: 'aws-waf'
+          action.target: 'network-traffic'
+          cloud.provider: 'aws'
+          cloud.service.name: 'waf'
+          observer.type: 'waf'
+
+          aws.waf.arn: '{{json_event.message.webaclId }}'
```

For semantic reason, I suggest renaming this field
```suggestion
          aws.waf.rule.arn: '{{json_event.message.webaclId }}'
```